atmo/jobs/admin.py

```diff
@@ -3,6 +3,27 @@
 from .models import SparkJob
 
 
+def run_now(modeladmin, request, queryset):
+    for job in queryset:
+        job.run()
+
+
 @admin.register(SparkJob)
 class SparkJobAdmin(admin.ModelAdmin):
-    pass
+    list_display = [
+        'identifier',
+        'size',
+        'created_by',
+        'start_date',
+        'end_date',
+        'last_run_date',
+        'is_enabled',
+        'most_recent_status',
+    ]
+    list_filter = [
+        'most_recent_status',
+        'size',
+        'is_enabled',
+    ]
+    search_fields = ['identifier', 'is_enabled', 'create_by__email', 'most_recent_status']
```

Comment:
Oh, not sure if `is_enabled` makes sense for `search_fields`. But maybe `current_run_jobflow_id`?